### DISCUSSION TITLE
Bottom tabs

### DISCUSSION THREAD

**Discussion by iBabu007** [TeamAmaze/AmazeFileManager#2781] [Ideas]
Please move down every tabs like
(cut, copy, paste, properties etc)
Nowadays every phone's screen is too large (6.5 inch)
So it is not handy.

**Comment by EmmanuelMess** [comment]
How would that work in Material design?

**Comment by VishalNehra** [comment]
@EmmanuelMess Even I'm wondering how we're gonna make this work out without making it something like Solid explorer copy

**Reply by EmmanuelMess** [reply]
I haven't used solid explorer. I don't know how similar this is to solid explorer.

**Comment by iBabu007** [comment]
But you also understand the problem. May be you also use the long screened phone.

**Comment by iBabu007** [comment]
No this is not a copy.
Everyone adopted good features .
like Android copy some ios features and ios also copy some android features it's very common thing.
You can add some more features like offline file sharing etc.
Lastly if possible then add it on fdroid it is the future

**Reply by shuvashish76** [reply]
> add some more features like offline file sharing etc.

https://forum.xda-developers.com/t/app-4-0-amaze-file-manager-material-theme.2937314/post-86734115

`7. Transfer files directly between two mobile devices on same wifi network using high speed peer to peer network`

@VishalNehra please make an announcement [here](https://github.com/TeamAmaze/AmazeFileManager/discussions/categories/announcements) as well.

**Comment by iBabu007** [comment]
You can add this features
Junk or cache file cleaner.
Built in app lock inside it.
Clone file finder.
Add storage consumption details per file
Storage analyser
Built in vault for images and files.
Offline file sharing like shareit.

**Comment by VishalNehra** [comment]
One thing at a time please.. we'll see whether bottom tabs are feasible design solution or not.

**Comment by VishalNehra** [comment]
I'm working on this in a custom branch

**Reply by EmmanuelMess** [reply]
Please create an issue and assign it to yourself, this is just to discuss the design details, not the implementation.

**Reply by VishalNehra** [reply]
https://github.com/TeamAmaze/AmazeFileManager/issues/3072
